python/ql/test/query-tests/Security/CWE-327-py2/InsecureProtocol.py

```diff
@@ -0,0 +1,52 @@
+import ssl
+from pyOpenSSL import SSL
```

Comment:
I don't see any mention of `pyOpenSSL` in the insecure default protocol... does that means that default values of pyOpenSSL is always safe?

Reply:
Yes, in that `SSL.SSLContext` requires a protocol argument; there is no default.